pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Description
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Dependency changes
+- [ ] Code refactor / cleanup
+- [ ] Documentation or wording changes
+- [ ] Other
+
+## Review Guide
+
+Note here any advice to reviewers. Does any aspect deserve special scrutiny?
+
+## Does this PR require tests?
+
+If so, please note what tests are included, and whether any still need work.
+
+- [ ] Unit tests
+- [ ] Integration tests
+- [ ] Fuzz tests
+
+## Checklist:
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have reviewed the diff to ensure it is legible, and there are no extraneous changes
+- [ ] I have run the linter (and didn't forget /pvt)
+- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
+- [ ] I have requested at least one review
+- [ ] If the base of this PR is not `master`, I've explained the dependency and merge plan
```

Comment:
```suggestion
- [ ] The diff is legible and has no extraneous changes
- [ ] Complex code has been commented, including external interfaces
- [ ] Tests are included for all code paths
- [ ] The base branch is either `master`, or there's a description of how to merge
```
I removed some that I consider unnecessary, and reworded the rest to avoid sounding too preachy

Reply:
Ok; if there are any "no" answers (e.g., no tests yet), or things that would have been covered in the review guide, it can all go in the description, as we do now.

Reply:
> Ok; if there are any "no" answers (e.g., no tests yet), or things that would have been covered in the review guide, it can all go in the description, as we do now.

- [ ] I have run the linter (and didn't forget /pvt)

Preachy? I was mainly preaching to myself :)